pipelines/New-pipeline/pipeline-tutorial-template.md

```diff
@@ -0,0 +1,109 @@
+# Protected Connected Index
+### Author(s): Jory Griffith
+#### Reviewed by:
+
+## Introduction
+
+The Protected Connected Index (ProtConn) is a component indicator in the Global Biodiversity Framework (GBF). ProtConn measures the percent of a given country or region that is conserved and managed through well-connected protected areas. This is an important indicator for assessing progress towards goals A and Target 3 of the GBF, which aim to have 30% of land area protected by a network of well-connected protected areas by 2030. ProtConn can be used to assess current progress towards these goals and for protected area planning and design.
```

Comment:
great introduction!

Wondering: can this be the intro of the pipeline as well? Currently the pipeline description does not say a lot, really. The pipeline doc remains more important than the readme.